Session-2.0.kt

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList
+
+fun main(args: Array<String>) {
+
+    val streetCap = 8; // Capacity of street
+    val countOfAllCars = 25; //Number of cars
+    var drivingCars: Int = 0 // Number of cars that want to drive
+    var allCars = ArrayList<Car>(); // Array of cars

```

Comment:
The code in `main` looks good. If you put it into separate class, you get the network that the task asks for. In that class, these variables might become fields.